packages/k2-alpine/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@avalabs/k2-alpine",
+  "license": "Limited Ecosystem License",
+  "version": "1.0.0",
+  "main": "expo/AppEntry.js",
+  "files": [
+    "./src/index.ts"
+  ],
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "lint": "eslint .",
+    "storybook-generate": "sb-rn-get-stories",
+    "tsc": "tsc -p ."
+  },
+  "dependencies": {
+    "dripsy": "^4.3.5",
+    "expo": "~51.0.28",
+    "expo-status-bar": "~1.12.1",
+    "react": "18.2.0",
+    "react-native": "0.74.5",
+    "react-native-reanimated": "^3.15.0"
+  },
+  "devDependencies": {
+    "@avalabs/tsconfig-mobile": "workspace:*",
+    "@babel/core": "^7.20.0",
+    "@react-native-async-storage/async-storage": "^1.24.0",
+    "@react-native-community/datetimepicker": "^8.2.0",
+    "@react-native-community/slider": "^4.5.2",
```

Comment:
these should live in dependencies

Reply:
All of these were added by Storybook. They can be moved inside of the app once they are being used by us.